### DISCUSSION TITLE
E-Fil+ Proposal: Data distribution rules established upfront by client

### DISCUSSION THREAD

**Discussion by kevzak** [filecoin-project/notary-governance#584] [E-Fil+ WG]
For discussion from the details of the main E-Fil+ issue: https://github.com/filecoin-project/notary-governance/issues/559

It is being proposed that for enterprise data: the data classification, distribution/ handling + rights will be established upfront by the client. 

Data types - sensitivity classification, together with regulation and compliance will need to be covered. 

The details and requirements of data rules are open for discussion. One enhancement to test during the pilot is an Enterprise web interface that will capture key data for solution design, contract, with API into GitHub.

**Comment by kevzak** [comment]
Additionally it is being proposed to consider less required copies for clients who don’t need widely distributed and are bound by sovereign reqs to keep data onshore

**Reply by xiaoliwe** [reply]
@kevzak hi, Kevin, would you like to share more about FIL+'s basic data rules(type, classification, etc.)? No more specific instructions were found, Thx.

**Reply by kevzak** [reply]
Hello @xiaoliwe thank you for the message. Here is a link to the current large data set requirements for Fil+ https://github.com/filecoin-project/filecoin-plus-large-datasets

In order for a client and their dataset to be eligible:

the dataset should be public, open, and mission aligned with Filecoin and Filecoin Plus. This also means that the data should be accessible to anyone in the network, without requiring any special permissions or access requirement

**Reply by kevzak** [reply]
For E-Fil+ we are looking to better enable private datasets to be usable on the network, this is the main difference from the current requirements. We need to ensure all data specifications are taken into consideration and managed between client and SP.

**Reply by xiaoliwe** [reply]
> 

Hi Kevin, Thanks so much ~

**Comment by kevzak** [comment]
Update: in recent Notary WG meetings the past few weeks, we have discussed requiring all data distribution plan details be agreed between client and SPs upfront and details will be required to included in the DataCap application. 

An LDN application medication was presented recently: https://github.com/filecoin-project/notary-governance/issues/593

Application details will now include:
-Data owner details
-Encryption details
-Geographic Storage location details (including if Geo-specific)
-SPs (miner IDs) to be involved
